src/sections/404/404.style.js

```diff
@@ -8,7 +8,7 @@ const L404SectionWrapper = styled.section`
 
     h1.message {
         font-size: 2rem;
-        line-height: 1.5rem;
+        line-height: 2rem;
     }
     h3.subtitle {      
         font-family: "Qanelas Soft Ultralight";        
```

Comment:
Any specific reason for this font?
If there is, please also add the generic family along with the font you are defining and for us that will be `sans-serif`. So, the CSS would be `font-family: 'Qanelas Soft Ultralight', sans-serif;`

Reply:
This font was used by Lee, so I didn't change it. Adding `sans-serif` to it.

Reply:
Removed it. Instead changed the font-weight

Reply:
Better!